src/components/organisms/UiMenu/_internal/UiMenuItem.vue

```diff
@@ -109,7 +109,8 @@ const defaultProps = computed(() => ({
 
     --button-font: #{functions.var($element + "-button", font, var(--font-body-1))};
     --button-letter-spacing: #{functions.var($element + "-button", letter-spacing, var(--letter-spacing-body-1))};
-
+    --border-radius-button: var(--border-radius-form);
```

Comment:
⚠️ You shouldn't change global variables in components. 
And you should use it before font variables ;) 
```suggestion
    --button-border-start-start-radius: var(--border-radius-form);
    --button-border-start-end-radius: var(--border-radius-form);
    --button-border-end-start-radius: var(--border-radius-form);
    --button-border-end-end-radius: var(--border-radius-form);
    --button-font: #{functions.var($element + "-button", font, var(--font-body-1))};
    --button-letter-spacing: #{functions.var($element + "-button", letter-spacing, var(--letter-spacing-body-1))};
```